src/lib.rs

```diff
@@ -250,6 +250,15 @@ appropriate `repr` attribute:
 use core::fmt::{Debug, Formatter, Result};
 use core::ops::*;
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub struct InvalidBits;
+
+impl core::fmt::Display for InvalidBits {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "Invalid bits")
+    }
+}
+
```

Comment:
@lkatalin `InvalidBits` will also need documentation.

Reply:
@npmccallum I added some documentation; let me know if it looks like it's missing anything expected.